kubeconfig

```diff
@@ -2,18 +2,67 @@ apiVersion: v1
 kind: Config
 clusters:
 - cluster:
-    certificate-authority: credentials/ca.pem
-    server: https://upp-staging-delivery-eu-api.ft.com
-  name: kube-aws-test-new-kube-aws-cluster
+    certificate-authority: credentials-prod-eu/ca.pem
```

Comment:
There is a small downside with this approach: you will be able to access just one cluster at any given time from multiple terminal sessions, because switching the kube context will write the kubeconfig file.
I think that for ops this would do, but I often need to access more than just one, so I use multiple kubeconfig files and change the KUBECONFIG env that stays set for a bash terminal. 
Also this is not really the same as in coco.

Reply:
Hmm, that's a good point, I hadn't considered that.

Perhaps we do want separate files then - I'll have a think about it.